simpleflow/swf/executor.py

```diff
@@ -1096,17 +1096,17 @@ def _workflow_id(self):
     def _run_id(self):
         return self._run_context.get('run_id')
 
-    def signal(self, name, workflow_id=None, run_id=None, propagate=True, *args, **kwargs):
+    def signal(self, name, *args, **kwargs):
         """
         Send a signal.
         :param name:
-        :param workflow_id:
-        :param run_id:
-        :param propagate:
         :param args:
         :param kwargs:
         :return:
         """
+        workflow_id = kwargs.pop('workflow_id', None)
```

Comment:
minor: could we keep a trace of these parameters in the doc? (I don't know the recommended way to do it. In the `:param kwargs:` line?)